### PR TITLE
Use pycodestyle instead of pep8

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**

According to the [package](https://pypi.org/project/pycodestyle/) info:
> This package used to be called pep8 but was renamed to pycodestyle to reduce confusion.

The change was already done in webots_ros2 and urdf2webots, but not here.

After the change, several other failures were detected, likely because the pep8 packages hasn't been updated since the change

### CODE REVIEW

File: resources/sumo_exporter/crossroad.py [hunk 1]
```diff
@@ -24,25 +24,20 @@ def __init__(self, crossroadType):
...
-        except:
-            self.rotation = [0.0, 0.0, 1.0, 0.0]
-        try:
-            self.connectedRoadIDs = [x.replace('"', '') for x in re.findall(r'connectedRoadIDs\s*\[([^\]]*)\]',
-                                                                            wbtString)[0].split()]
-        except:
-            pass
+
+        id = re.findall(r'id\s*"([^"]*)"', wbtString)
+        self.id = id[0] if not id else self.id
```
Comment (stefaniapedrazzi) [inline resources/sumo_exporter/crossroad.py:29]: This seems wrong, shouldn't be
```suggestion
        self.id = id[0] if id else self.id
```

The same wrong conditional is also used in the following lines for `translation`, `rotation`, `connectedRoadIDs`, `roadNumber` and `roadsWidth`.
And in the changes to `resources/sumo_exporter/road.py` file aiming at removing the `try/except` blocks.

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]

File: resources/sumo_exporter/crossroad.py (hunk 1 above)
Comment (ad-daniel) [reply resources/sumo_exporter/crossroad.py:29]: yes definitely, got it backwards :cry:

Comment (ad-daniel) [reply resources/sumo_exporter/crossroad.py:29]: ops, you're right I got it backwards

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you!
It seems that the tests didn't start, but everything seems fine now.

**Comment by ad-daniel** [comment]
when adding the test on the webots_ros2 repo it complained about the issues in the libcontroller. I noticed that we didn't check sources of the libcontroller on the webots side, since we have the new python API we should. I'll make a few tests and put it back to ready for review when done

File: lib/controller/python/controller/__init__.py [hunk 2]
```diff
@@ -47,3 +47,10 @@
 from controller.mouse import MouseState                        # noqa
 from controller.joystick import Joystick                       # noqa
 from controller.motion import Motion                           # noqa
+
+__all__ = [
+    Field, Node, ContactPoint, AnsiCodes, Accelerometer, Altimeter, Brake, Camera, CameraRecognitionObject, Compass, Connector,
+    Display, DistanceSensor, Emitter, GPS, Gyro, InertialUnit, LED, Lidar, LidarPoint, LightSensor, Motor, PositionSensor,
+    Radar, RadarTarget, RangeFinder, Receiver, Robot, Skin, Speaker, Supervisor, TouchSensor, Keyboard, Mouse, MouseState,
+    Joystick, Motion
```
Comment (stefaniapedrazzi) [inline lib/controller/python/controller/__init__.py:55]: What order did you use to list the classes?
I would rather use the alphabetical order to make this list cleaner.

**Review by stefaniapedrazzi — COMMENTED** [review]
Everything seems fine.
I just have one comment.

File: lib/controller/python/controller/__init__.py (hunk 2 above)
Comment (ad-daniel) [reply lib/controller/python/controller/__init__.py:55]: I've used the order as they appear in the imports above

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you!

**Pull request merged**